Dockerfile

```diff
@@ -11,7 +11,7 @@ ARG EXTRA_BUILD_APK_PACKAGES
 RUN apk add --no-cache --update \
     git tar build-base linux-headers autoconf automake libtool pkgconfig \
     dbus-dev bzip2 bison flex gmp-dev cmake lz4 libsodium-dev openssl-dev \
-    sed wget curl \
+    sed wget curl tpm2-tss-dev\
```

Comment:
These extra packages are supposed to go into the `EXTRA_BUILD_APK_PACKAGES` build-arg

Reply:
How I will be able to build TPM mapper library? erltss2. Is there any way to make "deps" in rebar.config conditional? And in app.src.
If it is possible, then it worth to exclude ECC code from RPM build